### PR TITLE
Update for 2018/02/15 release

### PR DESCRIPTION
**Pull request by steven-johnson**
(Note that recent versions of 'highlight' will barf if you don't include the 'Interpolation' value)

### CODE REVIEW

File: tutorials/tutorial_lesson_19_wrapper_funcs.html [hunk 1]
```diff
@@ -655,6 +655,10 @@
         img<span class="hl opt">.</span><span class="hl kwd">set</span><span class="hl opt">(</span>input<span class="hl opt">);</span>
         blur<span class="hl opt">.</span><span class="hl kwd">compile_jit</span><span class="hl opt">(</span>target<span class="hl opt">);</span>
         Buffer<span class="hl opt">&lt;</span><span class="hl kwb">int</span><span class="hl opt">&gt;</span> out <span class="hl opt">=</span> blur<span class="hl opt">.</span><span class="hl kwd">realize</span><span class="hl opt">(</span><span class="hl num">256</span><span class="hl opt">,</span> <span class="hl num">256</span><span class="hl opt">);</span>
+        <a onclick='return toggle("_311");'><b id=_311-show>// Click to show output ...</b></a><div id=_311 style= "display:none; background:#333333; color:#ffffff"><b>
+ > Error: OpenCL API not found: clGetPlatformIDs
```
Comment (abadams) [inline tutorials/tutorial_lesson_19_wrapper_funcs.html]: Not the best sample output :)

Comment (steven-johnson) [reply tutorials/tutorial_lesson_19_wrapper_funcs.html]: bahahaha :-)

Will update

**Comment by abadams** [comment]
Heh, the "once more" commit is labelled as from me. I'm guessing you did it in an existing checkout on a buildbot.

**Comment by steven-johnson** [comment]
Heh, yep, didn't think to check on that :-)

**Pull request merged**